purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyPurchasingData.kt

```diff
@@ -0,0 +1,17 @@
+package com.revenuecat.purchases.galaxy
+
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.PurchasingData
+import dev.drewhamilton.poko.Poko
+
+sealed class GalaxyPurchasingData : PurchasingData {
```

Comment:
Same as in the preview PR, might make sense to make these experimental... but NABD since we're merging to an integration branch